src/sentry/models/options/option.py

```diff
@@ -21,8 +20,29 @@ class Option(Model):  # type: ignore
     value = PickledObjectField()
     last_updated = models.DateTimeField(default=timezone.now)
 
+    class Meta:
+        abstract = True
+
+    __repr__ = sane_repr("key", "value")
+
+
+@region_silo_only_model
+class Option(BaseOption):
+    __include_in_export__ = True
+
     class Meta:
         app_label = "sentry"
         db_table = "sentry_option"
 
     __repr__ = sane_repr("key", "value")
+
+
+@control_silo_only_model
+class ControlOption(BaseOption):
+    __include_in_export__ = False
```

Comment:
Should probably be included in export like other options?

Reply:
You may be right.